scalafix-tests/src/test/scala/scalafix/tests/IntegrationPropertyTest.scala

```diff
@@ -190,7 +190,7 @@ class ScalaJs
           Command("examples/test:compile")
         )
       ),
-      skip = true // GenJsCode is hard: import renames + dependent types
+      skip = false // GenJsCode is hard: import renames + dependent types
```

Comment:
Did you run clean/compile after disabling scalafix? I think I disabled it for some reason while debugging.